src/main/java/sirius/biz/model/LoginData.java

```diff
@@ -424,7 +426,7 @@ public String getPasswordHash() {
         return passwordHash;
     }
 
-
+    @NoodleSandbox(NoodleSandbox.Accessibility.GRANTED)
```

Comment:
Is this too spicy?

Reply:
Maybe add an "getGeneratedPasswordIfDisplayed()" that returns the password only if "isDisplayGeneratedPassword()"? That should be sufficient for the use case, and adds a tiny bit security